src/stdlib_math.fypp

```diff
@@ -50,25 +52,32 @@ module stdlib_math
       end function ${RName}$
     #:endfor
 
-    ! Add support for complex linspace
-    #:for k1, t1 in CMPLX_KINDS_TYPES
-      #:set RName = rname("clinspace_default", 1, t1, k1)
+
+    ! Add suport for integer linspace    
+    !! Version: Experimental
+    !!
+    !! Create rank 1 array of linearly spaced elements from start to end.
+    !! If the number of elements is not specified, create an array with size 100. If n is a negative value,
+    !! return an array with size 0. If n = 1, return end. When dealing with integers as the `start` and `end` 
+    !! paramaters, the return type is always a double precision real.
+    #:for k1, t1 in INT_KINDS_TYPES
+      #:set RName = rname("linspace_default", 1, t1, k1)
       module function ${RName}$(start, end) result(res)
         ${t1}$, intent(in) :: start
-        ${t1}$, intent(in) :: end
-
-        ${t1}$ :: res(DEFAULT_LINSPACE_LENGTH)
+        ${t1}$, intent(in) :: end  
+  
+        real(dp) :: res(DEFAULT_LINSPACE_LENGTH)        
```

Comment:
Perhaps you can set your text editor to remove additional whitespace from lines. Line 67 is here a bogus commit. In Sublime for example you have the option `"trim_trailing_white_space_on_save": true`. VS Code has a similar [option](https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut).

Reply:
Gotcha, I'll poke around in the settings and try and get that taken care of.